WordPress/WordPressStatsWidgets/StatsWidgets.swift

```diff
@@ -7,5 +7,10 @@ struct WordPressStatsWidgets: WidgetBundle {
         WordPressHomeWidgetToday()
         WordPressHomeWidgetThisWeek()
         WordPressHomeWidgetAllTime()
+#if JETPACK_STATS_WIDGETS
+        if #available(iOS 16.0, *) {
```

Comment:
Another option could be using `FeatureFlag.lockScreenWidget`, and for now leave `FeatureFlag` with the value of `AppConfiguration.isJetpack`.

Reply:
I'd prefer using `AppConfiguration.isJetpack` here, it's more clear to me the widget is only for Jetpack.